src/players/shared/JwPlayerWrapper.tsx

```diff
@@ -61,6 +65,10 @@ const JwPlayerWrapper: React.FC<JwPlayerWrapperProps> = ({ playlist, playerUrl,
 					player: window.jwplayer(elementId),
 					ready: event,
 				});
+
+				if (onReady) {
```

Comment:
Does this delay the loading of the video player?

Reply:
No, at this point the player is "ready" and should be loaded. This is very similar to the current player's implementation.